dhall/tests/Dhall/Test/Import.hs

```diff
@@ -14,6 +14,7 @@ import qualified Control.Monad                    as Monad
 import qualified Control.Monad.Trans.State.Strict as State
 import qualified Data.Text                        as Text
 import qualified Data.Text.IO                     as Text.IO
+import Data.Void                                  (Void)
```

Comment:
Could you move this import up to the section with the other import lists, i.e. below the line

```
import Turtle (FilePath, (</>))
```